openedx/core/djangoapps/enrollments/views.py

```diff
@@ -987,3 +995,113 @@ def get_queryset(self):
         if usernames:
             queryset = queryset.filter(user__username__in=usernames)
         return queryset
+
+
+class EnrollmentAllowedView(APIView):
+    """
+    A view that allows the retrieval and creation of enrollment allowed for a given user email and course id.
+    """
+    authentication_classes = (
+        JwtAuthentication,
+    )
+    permission_classes = (IsStaffOrOwner,)
+    throttle_classes = (EnrollmentUserThrottle,)
```

Comment:
No action needed, just wanted to note that I looked up `EnrollmentUserThrottle` [[GitHub](https://github.com/eduNEXT/edx-platform/blob/776f4bf94e7f51542403f2122894ffdc051b817c/openedx/core/djangoapps/enrollments/views.py#L91)] and it has a more generous throttle for users with staff access, which presumably the user account used by server-to-server calls will have.